docs-src/develop.rst

```diff
@@ -140,8 +140,8 @@ For a full set of options, run::
 Building full data files
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-For instructions on how to build annotation (dganno) files and assembly
-(2bit) files, see `Genomes <genomes.html>`_.
+For instructions on how to build annotation (dganno) files, assembly
+(2bit) files, and Appris/MANE files, see `Genomes <genomes.html>`_.
```

Comment:
```suggestion
(2bit) files, and APPRIS/MANE files, see `Genomes <genomes.html>`_.
```

also not sure if this would be better (non html sphinx output is pretty rare though)
```
:doc:`genomes`
```